setup_python.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -ex
+
+case $DD_TARGET_ARCH in
+"x64")
+    CONDA_URL=https://repo.anaconda.com/miniconda/Miniconda3-${DD_CONDA_VERSION}-Linux-x86_64.sh
+    ;;
+"aarch64")
+    CONDA_URL=https://github.com/conda-forge/miniforge/releases/download/${DD_CONDA_VERSION}/Miniforge3-Linux-aarch64.sh
+    ;;
+*)
+    echo "Not installing conda"
+    exit 0
+esac
+
+curl -fsL -o ~/miniconda.sh $CONDA_URL
+bash ~/miniconda.sh -b
+rm ~/miniconda.sh
+
+PATH="${CONDA_PATH}/bin:${PATH}"
+conda init bash
+source /root/.bashrc
+
+# Setup pythons
+conda create -n ddpy2 python python=2
+conda create -n ddpy3 python python=3.8
+
+# Update pip, setuptools and misc deps
+conda activate ddpy2 \
+    && pip install -i https://pypi.python.org/simple pip==${DD_PIP_VERSION} \
+    && pip install --ignore-installed setuptools==${DD_SETUPTOOLS_VERSION} \
+    && pip install invoke distro==1.4.0 awscli==1.16.240
```

Comment:
`invoke` wasn't pinned before but I think we should pin it (on `datadog-agent` we use `1.4.1`):
```suggestion
    && pip install invoke==1.4.1 distro==1.4.0 awscli==1.16.240
```